README.md

```
@@ -1,95 +1,59 @@
 
 ### Implementation
 For Software:
 # Installation
-[commands]
+   npm install 
 
 # Run
-[commands]
+   npm run dev 
 
 ### Project Documentation
```

Markdownlint

Expected: h2; Actual: h3
Heading levels should only increment by one level at a time

(MD001, heading-increment)